index.d.ts

```diff
@@ -0,0 +1,52 @@
+import { AbortSignal } from "abort-controller"
+export type RandomnessBeacon = {
+    round: number
+    randomness: string
+    signature: string
+    previous_signature: string
+}
+
+export type NodeInfo = {
+    public_key: string
+    period: number
+    genesis_time: number
+    hash: string
```

Comment:
Is "hash" the "ChainHash"?

Reply:
yes - these keys are just what the go HTTP api returns